pkg/engagement/usecases/usecases.go

```diff
@@ -2,23 +2,30 @@ package usecases
 
 import (
 	"github.com/savannahghi/engagement/pkg/engagement/infrastructure"
+	"github.com/savannahghi/engagement/pkg/engagement/usecases/fcm"
 	"github.com/savannahghi/engagement/pkg/engagement/usecases/feed"
 )
 
 // Interactor is an implementation of the usecases interface
 type Interactor struct {
 	*feed.UseCaseImpl
 	*feed.NotificationImpl
+	*fcm.PushImpl
+	*fcm.FCMImpl
 }
 
 // NewInteractor initializes a new usecases interactor
 func NewInteractor(infrastructure infrastructure.Interactor) Interactor {
 
 	notification := feed.NewNotification(infrastructure)
 	feed := feed.NewFeed(infrastructure)
+	push := fcm.NewPush(infrastructure)
+	fcm := fcm.NewFCM(infrastructure)
 
 	return Interactor{
 		feed,
 		notification,
+		push,
+		fcm,
```

Comment:
Moving this work systematically like this also affords you the luxury of fixing "minor" things we over look like `good doc strings` and `good tests (unit, integration and acceptance).

For every service you move make sure it adheres to these.

Reply:
for sure. noted